pipelines/appco/scripts/longhorn_setup.sh

```diff
@@ -5,7 +5,11 @@ set -x
 export LONGHORN_REPO_BRANCH="${LONGHORN_VERSION}"
 
 source pipelines/utilities/kubectl_retry.sh
-source pipelines/utilities/run_longhorn_test.sh
+if [[ "${E2E_TEST}" == "true" ]]; then
```

Comment:
It would be better to keep this consistent with other pipelines. e.g., https://github.com/longhorn/longhorn-tests/blob/master/pipelines/fleet/scripts/longhorn-setup.sh#L14